### ISSUE TITLE
Get camera position in real time

### ISSUE THREAD

**Issue by worlderland** [potree/potree#1089]
We have integrated the Potree viewer in a React app and we are trying to display the camera position (x, y, z) in real time.

We are using the camera perspective and we found some values in viewer.scene.cameraP.position but it seems like these values are updated in real time, and in React they only get updated when the component is refreshed.

Anyone has an idea where we could find the real time position of the camera?

**Comment by HeadLessHUN** [comment]
The issue is in your React component not in three nor in potree.

**Comment by worlderland** [comment]
Hi @HeadLessHUN , thanks for your answer, even if it's not very helpful.

Let me rephrase the question... When we look at the object viewer.scene.getActiveCamera().position, are we looking in the right spot in order to read the current camera coordinates?
I have had this object console.logged on a timer (10ms) to see whether it was changing when I was changing the camera position, but it was not returning a real-time position update.

What would you suggest to read a real time value (if Potree has one). Once I get that, I'll fix my React component.

Thank you.

**Comment by HeadLessHUN** [comment]
that should return active camera position in realtime.

`viewer.scene.getActiveCamera().position
`
if it does not change then.somethings is really wrong. if you look at the sidebar implementation. then you can find something like this. but the previous one should work, because that is what requires to move around in three.

`viewer.scene.view.position`
